shared-scripts.sh

```diff
@@ -282,7 +282,7 @@ function useChildAccountCredentials {
         parent_acct=$(aws sts get-caller-identity | jq -cr '.Account')
         child_accts=$(aws organizations list-accounts | jq -c "[.Accounts[].Id | select(. != \"$parent_acct\")]")
         org_size=$(echo $child_accts | jq 'length')
-        opt_in_regions=$(jq -r '.[] | select(.optIn == true) | .name' ./scripts/e2e-test-regions.json)
+        opt_in_regions=$(jq -r '.[] | select(.optIn == true) | .name' $$CODEBUILD_SRC_DIR/scripts/e2e-test-regions.json)
```

Comment:
Did you intend this to be a double-dollar sign?
```suggestion
        opt_in_regions=$(jq -r '.[] | select(.optIn == true) | .name' $CODEBUILD_SRC_DIR/scripts/e2e-test-regions.json)
```